lib/lightning_web/live/credential_live/json_schema_body_component.ex

```diff
@@ -123,15 +123,7 @@ defmodule LightningWeb.CredentialLive.JsonSchemaBodyComponent do
         Required
       </span>
       <div class="col-span-2">
-        <%= apply(Phoenix.HTML.Form, @type, [
-          @form,
-          @field,
-          [
-            value: @value || "",
-            class: ~w(mt-1 focus:ring-primary-500 focus:border-primary-500 block
-               w-full shadow-sm sm:text-sm border-secondary-300 rounded-md)
-          ]
-        ]) %>
+        <.input type={@type} field={@form[@field]} value={@value || ""} />
```

Comment:
Nice! Is it possible to instead leverage the input components existing behaviour where it picks the value off the form field?

Reply:
Yeah, I tried that but for some reason, during editing it doesn't pick up that value